_data/strings.yml

```diff
@@ -1,2 +1,2 @@
-version: beta-20160929
-build_time: 2016/09/29 17:54:45 (go1.7.1)
+version: beta-20161006
+build_time: 2016/10/06 116:33:49 (go1.7.1)
```

Comment:
Doesn't your day have 116 hours? Fixing in separate PR. Pulled the trigger on this one already.